client/src/routes/settings/+page.svelte

```diff
@@ -48,8 +215,8 @@
       <Dialog bind:open={errPrompt} aria-labelledby="reauth-title" aria-describedby="err-msg-content">
         <Title id="reauth-title">NOTICE!</Title>
         <Content id="err-msg-content"
-          >Last sign in too long ago.
-          <br />Please Signin and Try Again</Content
+          >Last sign in too long ago. <br />
+          Please Signin and Try Again</Content
```

Comment:
make a more consistent user message

Reply:
@njumaah is this resolved now? if not can you give an example of what you think would be a better error message or explain what is inconsistent about the current message.